test/mocks/server/config_tracker.cc

```diff
@@ -2,14 +2,12 @@
 
 #include <string>
 
-#include "envoy/admin/v3/server_info.pb.h"
-#include "envoy/config/core/v3/base.pb.h"
-
-#include "common/singleton/manager_impl.h"
-
 #include "gmock/gmock.h"
 #include "gtest/gtest.h"
 
+using testing::_;
```

Comment:
These should be moved to inside the innermost namespace, and they should be fully-qualified. See https://abseil.io/tips/119

Reply:
fixed